multi-int-store/src/main/java/com/connexta/multiintstore/services/impl/IndexedMetadataDao.java

```diff
@@ -31,14 +30,8 @@ public IndexedMetadataDao(IndexedMetadataRepository repository) {
   }
 
   @Override
-  public void save(IndexedProductMetadata indexedProductMetadata)
-      throws StorageException, DuplicateIdException {
+  public void save(IndexedProductMetadata indexedProductMetadata) throws StorageException {
     Optional<IndexedProductMetadata> document = getById(indexedProductMetadata.getId());
```

Comment:
❓ document is no longer used, should it be deleted?

Reply:
This is why I tagged you 👍